packages/playwright-core/src/server/injected/roleUtils.ts

```diff
@@ -461,6 +461,38 @@ export function getElementAccessibleDescription(element: Element, includeHidden:
   return accessibleDescription;
 }
 
+export function getElementAccessibleErrorMessage(element: Element, includeHidden: boolean): string {
+  const cache = includeHidden ? cacheAccessibleErrorMessageHidden : cacheAccessibleErrorMessage;
+  let accessibleErrorMessage = cache?.get(element);
+
+  if (accessibleErrorMessage === undefined) {
+    accessibleErrorMessage = '';
+
+    const ariaInvalid = element.getAttribute('aria-invalid');
+    if (ariaInvalid === 'true') {
+      const errorMessageId = element.getAttribute('aria-errormessage');
+      if (errorMessageId) {
+        // Ensure the ID is valid (no whitespace)
+        if (!/\s+/.test(errorMessageId)) {
```

Comment:
According to the spec it can be an id reference list (i.e. multiple elements), we should handle that case too.

Reply:
You can use `getIdRefs()` helper.

Reply:
Thank you for the review! :)
 I've updated the code to handle `multiple aria-errormessage` references using `getIdRefs()`. Added a regression test to verify the behavior. Let me know if further changes are needed. :)